src/express/api.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+const axios = require(`axios`);
+
+const TIMEOUT = 1000;
+
+const port = process.env.API_PORT || 3000;
+const defaultUrl = `http://localhost:${port}/api/`;
+
+class API
+{
```

Comment:
```suggestion
class API {
```
в js так не пишут (исправь для всех методов тоже)